layers/targets/meta-raspberrypi/recipes-core/gateway-config/files/gateway-config.sh

```diff
@@ -256,6 +258,23 @@ do_select_au915_block() {
 }
 
 
+do_select_as923_block() {
+  # $1: concentratord version
+	# $2: model
+	# $3: model flags
+    FUN=$(dialog --title "Channel-plan configuration" --menu "Select the AS923 channel-block:" 15 60 8 \
+        1 "Channels  0 -  7 + 64" \
```

Comment:
@brocaar AS923 does not have like a channel name based in sub bands since channel and I don' want to set this too long... what do you think that we can approach this?

Reply:
![image](https://user-images.githubusercontent.com/165497/77294590-d7f93c00-6ce4-11ea-8de9-3a2e41ecf378.png)

Reply:
I understand now.